docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md

```diff
@@ -0,0 +1,72 @@
+# Connect to Nebula Graph
+
+Nebula Graph supports multiple types of clients, including a CLI client, a GUI client, and clients developed in popular programming languages. This topic provides an overview of Nebula Graph clients and a basic instruction on how to use the native CLI client, nebula-console.
+
+## Nebula Graph clients
+
+| Client                                                                 | Introduction                                           | Connection guide                                                                                                                                          |
+| ---------------------------------------------------------------------- | ------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [nebula-console](https://github.com/vesoft-inc/nebula-console)         | The native command-line interface of Nebula Graph      | [Connect to Nebula Graph with nebula-console](#use-nebula-console-to-connect-to-nebula-graph)                                                             |
+| [Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker) | The official graphical user interface for Nebula Graph | [Connect to Nebula Graph with Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker/blob/master/docs/nebula-graph-studio-user-guide-en.md) |
```

Comment:
Unify the description with Nebula Graph Studio. @oldLady344

Reply:
No need. Here is only a very brief description. The link is provided if anyone wants to know more.